draft-scenarios/campaigns/castle_two_knights_defense.tex

```diff
@@ -0,0 +1,243 @@
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling, \svgunit{silver} Dwelling, Citadel
+
+\subsection*{\MakeUppercase{AI Hero setup}}
+
+\textbf{Faction:} Necropolis
+
+\textbf{Enemy Army:} A Pack of Skeletons, a Pack of Zombies, a Pack of Liches, a Pack of Dread Knights
+
+\textbf{Enemy Deck:} 3 x Might cards, 3 x Magic Cards, 1 x Skill card
+
+\textbf{Enemy Spells:} 2 x Lightning Bolt, 1 x Curse
+
+\textbf{Enemy Skills:} Cloak of the Undead King IV and VI, the enemy will play whichever is possible at the time
+
+\subsection*{\MakeUppercase{Map setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout:
+
+\textbf{1 × Starting (I) Map Tile}
+\begin{itemize}
+    \item 1 x Necropolis (S1)
+\end{itemize}
+
+\textbf{2 × Far (II--III) Map Tile}
+\begin{itemize}
+    \item 2 x random Tiles from Necropolis or Dungeon (F1, F2, F4, F5, F7, F8)
+\end{itemize}
+
+\textbf{4 × Near (IV--V) Map Tile}
+\begin{itemize}
+    \item 2 x Necropolis (N1, N4)
+    \item 2 x Dungeon (N2, N5)
+\end{itemize}
+
+\textbf{1 × Center (VI--VII) Map Tile}
+\begin{itemize}
+  \item 1 × Dragon Utopia Tile (C1)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Heroes placement}}
+
+Place your Castle Hero on the center Field of the Necropolis start Tile S1.
+
+Place a Necropolis Hero on the bottom left Field of the leftmost Far Tile. 
+Reveal the Tile when the game begins.
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+
+\begin{itemize}
+  \item Defeat the enemy army
+\end{itemize}
+
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+\begin{itemize}
+  \item You lose one combat encounter
+  \item You run out of time at the end of the 8th Round
+\end{itemize}
+\end{multicols*}
+
+\newpage
+
+\begin{multicols}{2}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{1} Round:}
+\begin{itemize}
+  \item Read ``The Race'' section
+\end{itemize}
+
+\textbf{When you complete the Scenario:}
+\begin{itemize}
+  \item Read ``Checkmate'' section
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional rules}}
+
+During this Castle campaign Scenario, the following rules apply:
+
+\begin{itemize}
+  \item The Glory of Erathia building is unavailable
+  \item Each time you visit a Trading Post, draw the top cards from the \svgunit{bronze} bronze, \svgunit{silver} silver, and \svgunit{golden} golden neutral enemy Decks.
+  You may recruit any, or all of them by paying their cost.
+  Shuffle any un-hired ones back into their Decks
+  \item After defeating the neutral army at the Dragon Utopia, add a random dragon from the \svgunit{azure} Azure neutral Deck to your army
+  \item The enemy army follows the normal AI movement rules, stopping to capture Settlements and Mines, with its ultimate goal to reach the Dragon Utopia
+  \item If the enemy army reaches the Dragon Utopia before you do, flip the Pack side of Dread Knights to Few and add a Pack of Ghost Dragons
+\end{itemize}
+
+\begin{tikzpicture}[overlay]
+    \node at (-1.0, -9.0) {
```

Comment:
Please move it slightly up, so that the spacing between the text and the bottom graphic is equal-ish.